datadog/resource_datadog_synthetics_test_test.go

```diff
@@ -49,6 +73,8 @@ func TestAccDatadogSyntheticsSSLTest_importBasic(t *testing.T) {
 				ResourceName:      "datadog_synthetics_test.ssl",
 				ImportState:       true,
 				ImportStateVerify: true,
+				// Assertions will be imported into the new schema by default, but we can ignore them as users need to update the local config in this case
+				ImportStateVerifyIgnore: []string{"assertions", "assertion"},
```

Comment:
Is the issue here that on import we will have both `assertions` and `assertion` in the state? Is this going to cause an unempty diff on future `apply`s? 

Actually, re-reading some of the code above, it looks like you check for the config before saving this field, in `updateSyntheticsTestLocalState`, so is this ignore still a requirement?

Reply:
I check the config before saving the field, but it only works if I already have state. IIUC, the `ImportStateVerify` compares to a plain import, which doesn't compare to anything locally.